content/sensu-go/6.2/operations/deploy-sensu/secure-sensu.md

```diff
@@ -216,16 +216,16 @@ To enable agent mTLS authentication:
 
 3. After you create backend and agent certificates, modify the backend configuration:
 {{< code yml >}}
-agent-auth-cert-file: "/path/to/backend-1.pem"
-agent-auth-key-file: "/path/to/backend-1-key.pem"
-agent-auth-trusted-ca-file: "/path/to/ca.pem"
+agent-auth-cert-file: "/etc/sensu/tls/agent.pem"
+agent-auth-key-file: "/etc/sensu/tls/agent-key.pem"
+agent-auth-trusted-ca-file: "/etc/sensu/tls/ca.pem"
```

Comment:
There is only one `ca.pem` listed in the examples in Generate certificates. See the directory example for /etc/sensu/tls on backend-1 under https://docs.sensu.io/sensu-go/latest/operations/deploy-sensu/generate-certificates/#generate-agent-certificate to see what I mean.

As a result, I am not sure whether the values should be different for `agent-auth-trusted-ca-file` and `trusted-ca-file` (below in line 228).